traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 8.7.2
+version: 8.7.3
```

Comment:
Should be a minor bump and not a patch bump, no?

Reply:
yup